### PR TITLE
[Merged by Bors] - refactor(Data/Int/Bitwise): use `<<<` and `>>>` notation

### PR DESCRIPTION
**Pull request by eric-wieser**
The lemma names have changed to use `shiftLeft` and `shiftRight` to match the new `Nat` names.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Int/Bitwise.lean [hunk 1]
```diff
@@ -365,96 +365,100 @@ theorem testBit_lnot : ∀ n k, testBit (lnot n) k = not (testBit n k)
 #align int.test_bit_lnot Int.testBit_lnot
 
 @[simp]
-theorem shiftl_neg (m n : ℤ) : shiftl m (-n) = shiftr m n :=
+theorem shiftLeft_neg (m n : ℤ) : m <<< (-n) = m >>> n :=
```
Comment (ADedecker) [inline Mathlib/Data/Int/Bitwise.lean:368]: Are the parentheses still necessary?

File: Mathlib/Data/Int/Bitwise.lean [hunk 2]
```diff
@@ -365,96 +365,100 @@ theorem testBit_lnot : ∀ n k, testBit (lnot n) k = not (testBit n k)
...
+    subNatNat_elim n k.succ (fun n k i => (↑m) <<< i = (Nat.shiftLeft' false m n) >>> k)
       (fun (i n : ℕ) =>
         by dsimp; simp [- Nat.shiftLeft_eq, ← Nat.shiftLeft_sub _ , add_tsub_cancel_left])
-      fun i n =>
-        by dsimp; simp [- Nat.shiftLeft_eq, Nat.shiftLeft_zero, Nat.shiftRight_add,
-                        ← Nat.shiftLeft_sub, shiftl]
+      fun i n => by
+        dsimp
+        simp [- Nat.shiftLeft_eq, Nat.shiftLeft_zero, Nat.shiftRight_add,
+                        ← Nat.shiftLeft_sub]
```
Comment (ADedecker) [inline Mathlib/Data/Int/Bitwise.lean:420]: This indentation is very weird.

File: Mathlib/Data/Int/Bitwise.lean (hunk 1 above)
Comment (eric-wieser) [reply Mathlib/Data/Int/Bitwise.lean:368]: Surprisingly, yes!

File: Mathlib/Data/Int/Bitwise.lean (hunk 2 above)
Comment (eric-wieser) [reply Mathlib/Data/Int/Bitwise.lean]: Fixed, I forgot to un-indent when un-line-wrapping.

**Comment by ADedecker** [comment]
Thanks!

bors merge

**Pull request closed without merging**